tests/resource/resource_test_helper.py

```diff
@@ -1,57 +1,116 @@
+"""
+Fixtures and helper functions to test models and test resources
+"""
+from __future__ import (absolute_import, division,
+                        print_function, unicode_literals)
+from builtins import *
+
 from collections import namedtuple
 
 from sqlalchemy.ext.declarative.api import declarative_base
+from sqlalchemy.engine.base import Engine
+from sqlalchemy.sql.schema import Column
+from sqlalchemy.sql.sqltypes import Integer, String
+from sqlalchemy import ForeignKey
+from sqlalchemy.orm import relationship
+
 from zsl.db.model.app_model import AppModel
 from zsl.db.model.raw_model import ModelBase
-from sqlalchemy.sql.sqltypes import VARCHAR, INTEGER
-from sqlalchemy.sql.schema import Column
 from zsl.utils.injection_helper import inject
-from sqlalchemy.engine.base import Engine
 from zsl.application.modules.alchemy_module import SessionHolder
 
 test_settings = {
     'DATABASE_URI': 'sqlite:///:memory:',
-    'DATABASE_ENGINE_PROPS': {'echo': True},
+    'DATABASE_ENGINE_PROPS': {},
+    'JSON_AS_ASCII': False
 }
 
 Base = declarative_base()
 
 
-class DummyAppModel(AppModel):
+class UserAppModel(AppModel):
     pass
 
 
-class DummyModel(Base, ModelBase):
-    __tablename__ = 'dummy'
+class UserModel(Base, ModelBase):
+    __tablename__ = 'users'
+    _app_model_class = UserAppModel
 
-    _app_model_class = DummyAppModel
+    id = Column('id', Integer, primary_key=True, nullable=False)
+    name = Column('name', String, nullable=False)
 
-    id = Column('id', INTEGER(), primary_key=True, nullable=False)
-    val = Column('val', VARCHAR(length=15), nullable=False)
+    def __repr__(self):
+        return '<User(id=\'{}\',name=\'{}\')>'.format(self.id, self.name)

```

Comment:
I renamed DummyModel to UserModel, it is easier to build upon something concrete e.g. some related model.